packages/jest-preset-mc-app/setup-test-framework.js

```diff
@@ -1,15 +1,22 @@
 require('unfetch/polyfill');
 require('jest-enzyme');
 require('@testing-library/jest-dom/extend-expect');
+require('./polyfills/intl');
+
 const Enzyme = require('enzyme');
 const Adapter = require('enzyme-adapter-react-16');
 const ShallowWrapper = require('enzyme/ShallowWrapper');
+import { configure as configureRtl } from '@testing-library/react';
 const configureEnzymeExtensions = require('@commercetools/enzyme-extensions');
 const commerceToolsEnzymeMatchers = require('@commercetools/jest-enzyme-matchers');
-require('./polyfills/intl');
+const loadConfig = require('./load-config');
+
+const jestConfig = loadConfig();
 
 Enzyme.configure({ adapter: new Adapter(), disableLifecycleMethods: true });
 
+configureRtl(jestConfig.rtlConfig);
```

Comment:
Naming what we can specify `rtlConfig`. Not the biggest fan of abbriviations but we know what `rtl` stands for and also export `rtlRender` from `test-utils`.